apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -47,11 +47,16 @@ const App = (): JSX.Element => {
 				<>
 					<h2>Users:</h2>
 					<h3>Status: {dataStatus}</h3>
-					<ul>
-						{users.map((user) => (
-							<li key={user.id}>{user.email}</li>
-						))}
-					</ul>
+
+					{dataStatus === "pending" ? (
```

Comment:
move this to a constant
```suggestion
					{isLoading ? (
```